compass/landice/tests/ensemble_generator/ensemble/namelist.landice

```diff
@@ -35,13 +35,13 @@
     config_adaptive_timestep = .true.
     config_adaptive_timestep_calvingCFL_fraction = 0.8
     config_adaptive_timestep_include_calving = .true.
-    config_adaptive_timestep_CFL_fraction = 0.8
+    config_adaptive_timestep_CFL_fraction = 0.2
```

Comment:
I'm not sure what the best choice here is.  But given we are inevitably stressing the model with the parameter range considered, erring on the small side is probably better.  On the flip side, the ensemble ends up costing a lot more than it otherwise would need to be for most runs, which really adds up.

Reply:
Could you add a meta adaptive time step that changes this number based on some assessment of the riskiness of a simulation? ;-)

Reply:
Ha, I actually thought about that.  Or have it try with an aggressive value, and if that fails, try a second time with a much more conservative value.  That seems possible, but I decided to stick with the principal of having computers do more work than humans for now and save this for a possible later refinement.  I'll add a note in the user guide about this issue.

Reply:
I decided to make the CFL fraction a cfg option to force a user to consciously select this value, given a poor choice could make the ensemble 2 or more times more expensive than it needs to be.  Addressed in 1973b72a111cb30fdb84d54f3e3cb4b6d46a202c

Reply:
>  but I decided to stick with the principal of having computers do more work than humans for now and save this for a possible later refinement.

Ask ChatGPT to write the code for you.